src/test/java/seedu/address/testutil/PropertyBuilder.java

```diff
@@ -22,14 +25,14 @@ public class PropertyBuilder {
     public static final String DEFAULT_ADDRESS = "333 Thompson Road";
     public static final String DEFAULT_DESCRIPTION = "A 5-storey condo on top of a hill - "
             + "Peak Residence offers you serenity away from the hustle and bustle with breathtaking views all around.";
-    public static final String DEFAULT_SELLER = "Aaron";
+    public static final Owner DEFAULT_OWNER = new Owner(new Name("John Doe"), new Phone("80001000"));
```

Comment:
You can use the `DEFAULT_NAME` and `DEFAULT_PHONE` also

Reply:
good catch, i'll add those static values